src/controllers/subscriptionController.js

```diff
@@ -40,7 +40,7 @@ module.exports = {
 
   editSubscription: async (req, res) => {
     try {
-      await subsRepo.editSubscriptionByAccountId(req.body.accountId, req.body.subscription);
+      await subsRepo.editSubscriptionByAccountId(req.params.accountId, req.body);
```

Comment:
Thats correct, the accountID is passed in the url anyway so writing it in the body is not nessecary.